### ISSUE TITLE
File not found on Linux

### ISSUE THREAD

**Issue by james-hu** [jeffpriz/vsts-json-to-variable#1]
When trying to use it on a Linux agent, I got this error:
`
2018-04-16T02:59:15.0297019Z ##[section]Starting: JSON to Variable
2018-04-16T02:59:15.0353449Z ==============================================================================
2018-04-16T02:59:15.0366886Z Task         : Json 2 Variable
2018-04-16T02:59:15.0380248Z Description  : Task that will read a JSON file and create build and/or release variables available to other tasks for the data in the JSON
2018-04-16T02:59:15.0394682Z Version      : 1.0.3
2018-04-16T02:59:15.0407722Z Author       : Jeff Przylucki
2018-04-16T02:59:15.0420937Z Help         : 
2018-04-16T02:59:15.0434745Z ==============================================================================
2018-04-16T02:59:15.3692313Z ##[error]File not found: '/home/vsts_agent2/_work/_tasks/oneLuckiDevJson2Variable_6f9cdff3-a6b9-47e1-90a8-6590ba11e157/1.0.3/./dist/Json2Variable.js'
2018-04-16T02:59:15.3751431Z ##[section]Finishing: JSON to Variable
`

Probably about path naming or packaging?

**Comment by james-hu** [comment]
I think it is about file name case sensitive. The actual file as can be found in the repo is in lower case `json2variable.js` while the file referred to is in camel case `Json2Variable.js`

**Comment by jeffpriz** [comment]
Thank for the feedback I'll get a push out shortly with that corrected. 

What Agent version are you running?

thanks

Jeff

**Comment by jeffpriz** [comment]
Published a fix and pushed it to the Marketplace.  You should be able to install the latest version (1.0.5) from the VS Marketplace now. 
Please let me know if that resolves your troubles.

thanks
Jeff

**Comment by james-hu** [comment]
Thanks Jeff! Just tested with the hosted Linux agent (preview), the version info shows it is 1.0.4. It works... however, the value has quotes at the start and the end of the string.

`
==============================================================================
Task         : Json 2 Variable
Description  : Task that will read a JSON file and create build and/or release variables available to other tasks for the data in the JSON
Version      : 1.0.4
Author       : Jeff Przylucki
Help         : 
==============================================================================
`

The file content is:
`
{
 "myVar1": "123",
 "myVar2": "this is the value"
}
`

I got this from an "env" command:
`
TEST_MYVAR1="123"
TEST_MYVAR2="this is the value"
`
which I think should be
`
TEST_MYVAR1=123
TEST_MYVAR2=this is the value
` instead.

**Comment by jeffpriz** [comment]
James, thanks for confirming that the fix works. I'm going to close this issue because of that fix. 

I also took a peek at how the task is handling string variables and did indeed find an issue with out it is placing the variable value. I now strip "'s from the start and end of the string if they are present.  I've verified that those string variables are functioning as-expected on my VSTS account.  Thank you for that report.

There is now v1.0.6 in the marketplace to address that trouble.
